tests/test_black.py

```diff
@@ -1168,7 +1168,7 @@ def test_reformat_one_with_stdin_and_existing_path(self) -> None:
             report = MagicMock()
             # Even with an existing file, since we are forcing stdin, black
             # should output to stdout and not modify the file inplace
-            p = Path(str(THIS_DIR / "data/collections.py"))
+            p = Path(str(THIS_DIR / "data" / "simple_cases" / "collections.py"))
```

Comment:
```suggestion
            p = THIS_DIR / "data" / "simple_cases" / "collections.py"
```
I know this is not your doing, but I'm pretty sure we don't need a double conversion if `THIS_DIR` is a path already 😅